### ISSUE TITLE
Sometimes silences page has "'totalSilences' of undefined" error (v0.4.0)

### ISSUE THREAD

**Issue by cofyc** [prometheus/alertmanager#477]
errors logs in chrome console:

```
angular.min.js:107 TypeError: Cannot read property 'totalSilences' of undefined
    at new <anonymous> (https://xxxx/app/js/app.js:270:36)
    at e (https://xxxx/lib/angular.min.js:39:193)
    at Object.instantiate (https://xxxx/lib/angular.min.js:39:310)
    at https://xxxx/lib/angular.min.js:80:313
    at link (https://xxxx/lib/angular-route.min.js:7:268)
    at aa (https://xxxx/lib/angular.min.js:73:90)
    at K (https://xxxx/lib/angular.min.js:62:39)
    at g (https://xxxx/lib/angular.min.js:54:410)
    at https://xxxxlib/angular.min.js:53:480
    at https://xxxx/lib/angular.min.js:55:397 <div ng-view="" class="ng-scope">
```

**Comment by grobie** [comment]
Thanks, this is fixed in the latest commit in the `release-0.4` branch. I'll release a `0.4.1` release today. Sadly, more thinks are not ideal there.

**Comment by grobie** [comment]
Fixed in [v0.4.1](https://github.com/prometheus/alertmanager/releases/tag/v0.4.1).
